daemon/daemon_unix.go

```diff
@@ -290,6 +293,34 @@ func (daemon *Daemon) adaptContainerSettings(hostConfig *containertypes.HostConf
 	return nil
 }
 
+// When a container shares its namespace by container name,
```

Comment:
Can you change the comment to `adaptSharedNamespaceContainer replaces a container name with its container ID when ...` or something similar.